st/clitests/auth_spec_negative_and_fi.py

```diff
@@ -773,4 +773,4 @@ def get_federation_token_test():
 config['cacheTimeout'] = "30"
 config.write()
 os.system('systemctl restart s3authserver')
-time.sleep(30)
\ No newline at end of file
+time.sleep(30)
```

Comment:
This should be reverted because the file is unedited and unrelated to this change.

Reply:
I see no difference in file st/clitests/auth_spec_negative_and_fi.py b/w branch object-versioning vs put_bucket_versioning_final